vocabularies-gsq/gsq-alias.ttl

```diff
@@ -311,6 +344,7 @@ cs:
         als:rshq ,
         als:service-provider ,
         als:treasury ,
+        als:wims-sample-id ,
```

Comment:
GSQ R and M number concepts should be included here as well, shouldn't they?

Reply:
There is a GSQ legacy concept, which doesn't appear here, so these appear to be other government organisations. GSQR and GSQM are narrower concepts of GSQW legacy.